src/components/share-structure/ShareStructure.vue

```diff
@@ -903,7 +991,7 @@ export default class ShareStructure extends Vue {
    */
   protected resetData (): void {
     function scrollToTop (element: any): void {
-      const isJestRunning = (process.env.JEST_WORKER_ID !== undefined)
+      const isJestRunning = (import.meta.env.JEST_WORKER_ID !== undefined)
```

Comment:
Hopefully this will still be OK if this component is imported into a VueCLI project but I don't know. My plan is to up-version at least this component in case we need to patch the non-Vite version.

Reply:
Sev, while I was working on Vite migration in Create UI, console was telling me that process is undefined for the common components. So, I'm assuming the opposite also applies.

Reply:
^^ Actually, I'm releasing this component as v2.2.0 (prev version was 2.1.23).